피보나치.js

```diff
@@ -0,0 +1,13 @@
+let fs = require('fs');
+let input = fs.readFileSync('/dev/stdin').toString().trim()//trim => 공백제거
+let n = parseInt(input);
+
+const obj = {0: 0, 1: 1}; //메모이제이션
+
+function fibo(el) {
+    if (obj[el] !== undefined) return obj[el];
```

Comment:
```suggestion
  if (el in obj) return obj[el];
```
undefined가 없는 것을 나타내주는 것은 맞지만, object에 있는지 포함 여부 조건으로 사용하는 것은 나중에 다른 문제를 풀 때 정확한 정보를 얻을 수 없을 것 같아 다른 방법도 적어드려요 키값 in object를 이용하면 키값의 포함 여부가 불리안 값으로 나옵니다.  이를 통해 푸는 것도 괜찮은 것 같아요. 

메모이제이션이란 것을 몰랐는데 덕분에 배워갑니다. 여러문제에서 사용 가능할 것 같아요 :)